python/py-jellyfish/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        jamesturk jellyfish 0.8.2
+fetch.type          git
+revision            0
+name                py-${github.project}
+
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         A library for doing approximate and phonetic matching\
+                    of strings.
+long_description    ${description}
+
+homepage            http://github.com/jamesturk/jellyfish
+
+checksums           rmd160  4bc50c79e582c6336d5c5f16975da3d180dad2de \
+                    sha256  3ec7a1f16fcde01f6591bd9dc93df2613d440c07b2d5642adb423988eefd481c \
+                    size    21092
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_test-append \
+                    port:py${python.version}-tox
+
+    post-fetch {
+         system -W ${worksrcpath} "git submodule update --init --recursive"
+    }
+
+    test.run        yes
+    test.cmd        tox-${python.branch}
```

Comment:
upstream runs the test as `py.test jellyfish/test.py`; so please remove the `tox` stuff, that's not needed and add a test-dependency on `py-pytest`.

Reply:
The repo says explicitly it wants `tox` for testing:

https://github.com/jamesturk/jellyfish#running-tests

Reply:
but for testing on [Travis](https://github.com/jamesturk/jellyfish/blob/master/.travis.yml) they don't use it and just call py.test directly